### PR TITLE
upgpkg: spotify 1:1.2.20.1210-3: fix quoting for custom flags

### PR DESCRIPTION
**Pull request by christian-heusel**
cc @Antiz96

### CODE REVIEW

File: spotify/spotify.sh [hunk 1]
```diff
@@ -5,9 +5,9 @@ XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
...
 if [[ -f "${XDG_CONFIG_HOME}/spotify-flags.conf" ]]; then
-    SPOTIFY_USER_FLAGS="$(grep -v '^#' "${XDG_CONFIG_HOME}/spotify-flags.conf")"
-    echo "User flags: ${SPOTIFY_USER_FLAGS}"
+    mapfile -t SPOTIFY_USER_FLAGS <<< "$(grep -v '^#' "${XDG_CONFIG_HOME}/spotify-flags.conf")"
+    echo "User flags:" "${SPOTIFY_USER_FLAGS[@]}"
 fi
 
-# Launch Spotify
-exec /opt/spotify/spotify "${SPOTIFY_USER_FLAGS}" "${@}"
+# Launch Spotify with the given flags
```
Comment (Antiz96) [inline spotify/spotify.sh:12]: That's just a detail, but there might be no given flags actually. 
It's just a comment though, no big deal ^^

File: spotify/spotify.sh [hunk 2]
```diff
@@ -5,9 +5,9 @@ XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
...
+exec /opt/spotify/spotify "${SPOTIFY_USER_FLAGS[@]}" "$@"
```
Comment (Antiz96) [inline spotify/spotify.sh:13]:
```suggestion
exec /opt/spotify/spotify "${SPOTIFY_USER_FLAGS[@]}" "${@}"
```

Comment (christian-heusel) [reply spotify/spotify.sh:13]: Yeah thats how it previously was, but I dont like it for `$@` 😆

File: spotify/spotify.sh (hunk 1 above)
Comment (christian-heusel) [reply spotify/spotify.sh:12]: I didnt find a better wording so I'll just keep it that way ^^

**Pull request merged**